src/GrumPHP/Task/PhpVersion.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Runner\TaskResultInterface;
+use GrumPHP\Task\Context\ContextInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * PhpVersion task
+ */
+class PhpVersion implements TaskInterface
+{
+
+    /**
+     * @var array
+     */
+    private $versions;
+
+    /**
+     * @var string
+     */
+    private $projectVersion;
+
+    /**
+     * PhpVersion constructor.
+     * @param GrumPHP $config
+     * @param array $versions
+     */
+    public function __construct(GrumPHP $config, array $versions)
+    {
+        $this->versions = $versions;
+        $options = $config->getTaskConfiguration($this->getName());
+        if (array_key_exists('project', $options)) {
+            $this->projectVersion = $options['project'];
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'phpversion';
+    }
+
+    /**
+     * @return array
+     */
+    public function getConfiguration()
+    {
+        return [];
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(
+            [
+                'project' => null,
+            ]
+        );
+
+        $resolver->addAllowedTypes('project', ['null', 'string']);
+
+        return $resolver;
+    }
+
+    /**
+     * This methods specifies if a task can run in a specific context.
+     *
+     * @param ContextInterface $context
+     *
+     * @return bool
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return true;
+    }
+
+    /**
+     * @param ContextInterface $context
+     *
+     * @return TaskResultInterface
+     */
+    public function run(ContextInterface $context)
```

Comment:
This code isn't testable.
Maybe it's better to move it to a separate class and test all the different cases in that class?
Will it work for people who are using nightly builds and pre-releases?